cltk/corpus/swadesh.py

```diff
@@ -22,8 +22,10 @@
 
+swadesh_ar = [' أنا‎',' أنت‎, أنت‎',' هو‎, هي‎',' نحن‎',' أنتم‎,‎ أنتن‎,‎ أنتما‎',' هم‎,‎ هن‎,‎ هما‎',' هذا‎',' ذلك‎',' هنا‎',' هناك‎','  من‎','  ما‎, ماذا‎',' أين‎',' متى‎',' كيف‎',' لا‎,‎‎ لم‎,‎ ما‎',' كل‎','  كثير‎','  كم‎','  قليل‎','  آخر‎',' واحد‎','‏ ‏اثنان‎','  ثلاثة‎',' أربعة‎',' خمسة‎','  كبير‎','  طويل‎','  رحب‎,‎ واسع‎',' سميك‎','  ثقيل‎','  صغير‎','  قصير‎','  ضيق‎',' رقيق‎','‏   ‏امرأة‎','  رجل‎',' إنسان‎',' طفل‎',' زوجة‎','  بعل‎',' أم‎','  أب‎','  حيوان‎',' سمك‎',' طير‎',' كلب‎',' قملة‎','  أفعى‎, ثعبان‎','  دودة‎','  شجرة‎','  غابة‎','  عود‎,‎‎ حطب‎',' فاكهة‎',' زرع‎',' ورقة‎','  جذر‎',' لحاء‎, قلف‎','  زهرة‎','  عشب‎',' حبل‎',' جلد‎',' لحم‎',' دم‎','  عظم‎',' سمن‎',' بيض‎',' قرن‎',' ذَنَب‎',' ريشة‎','  شعر‎',' رأس‎',' أذن‎',' عين‎',' أنف‎',' فم‎','  سن‎','  لسان‎','  ظفر‎',' قدم‎',' ساق‎ ','  ركبة‎','  يد‎','  جَنَاح‎','  بطن‎',' أمعاء‎',' عنق‎',' ظهر‎',' صدر‎',' قلب‎',' كبد‎',' شرب‎','   أكل‎',' عض‎','  مص‎','  بصق‎',' تقيأ‎','  نفخ‎',' تنفس‎','  ضحك‎',' رأى‎',' سمع‎',' علم‎',' اعتقد‎',' شم‎','  خاف‎',' نام‎',' عاش‎',' مات‎',' قتل‎',' قاتل‎','  صاد‎',' ضرب‎','   قطع‎',' قَسَمَ‎ ',' طَعَنَ‎','  خدش‎',' حفر‎',' سبح‎',' طار‎',' مشى‎',' أتى‎',' استلقى‎','  جلس‎',' قام‎',' دار‎',' سقط‎',' أعطى‎','  أخذ‎',' عصر‎',' حف‎','  غسل‎',' محا‎',' جر‎','  دفع‎',' رمى‎',' ربط‎',' قطب‎',' عد‎','  قال‎',' غنى‎',' لعب‎',' طفا‎',' سال‎',' أجلد‎','  انتفخ‎',' شمس‎',' قمر‎',' نجم‎',' ماء‎',' مطر‎',' نهر‎',' بركة‎','  بحر‎',' ملح‎',' صخر‎',' رمل‎',' غبار‎','  أرض‎',' سحابة‎',' ضباب‎','  سماء‎','  هواء‎','  ثلج‎',' جليد‎','  دخان‎','  نار‎',' رماد‎','  شعل‎,‎ حرق‎','  طريق‎','  جبل‎',' أحمر‎','  أخضر‎ ',' أصفر‎','  أبيض‎','  أسود‎','  ليلة‎','  يوم‎',' سنة‎',' دفيء‎','  بارد‎','  ممتلئ‎',' جديد‎','  قديم‎','  جيد‎',' سيئ‎',' فاسد‎','  وسخ‎',' مستقيم‎','  مستدير‎','  حاد‎',' كليل‎','  ناعم‎','  رطب‎',' جاف‎',' صحيح‎','  قريب‎','  بعيد‎','  يمين‎','  يسار‎','  بـ‎ ',' في‎','  مع‎,‎ بـ‎','  و‎',' إن‎ ,‎ إذا‎,‎ لو‎','  لأن‎,‎ بسبب‎',' اسم‎']
```

Comment:
It seems like this list [Arabic Swadesh list](https://en.wiktionary.org/wiki/Appendix:Arabic_Swadesh_list) that i found in wikt.
Thank you for the good work 👍 .

Reply:
use `""` instead of `''`

Reply:
Remove MAS words, Just keep CA words